eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -333,10 +334,16 @@ jobs:
       # Generate Documentation
       - pwsh: |
           $setupFile="doxygen-1.8.15-setup.exe";
-          Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://azuresdktooling.blob.core.windows.net/dependencies/doxygen-1.8.15-setup.exe" `
-          -OutFile $setupFile | Wait-Process; Start-Process -Filepath .\$setupFile -ArgumentList @("/VERYSILENT") -Wait
+          Invoke-WebRequest `
+            -MaximumRetryCount 10
+            -Uri "https://azuresdktooling.blob.core.windows.net/dependencies/doxygen-1.8.15-setup.exe" `
+            -OutFile $setupFile `
+          | Wait-Process;
+
+          Start-Process -Filepath .\$setupFile -ArgumentList @("/VERYSILENT") -Wait
         workingDirectory: $(Agent.TempDirectory)
         displayName: Download and Install Doxygen
+        timeoutInMinutes: 5
```

Comment:
Can you explain what motivated this change and also why we use 10 minutes above and 5 minutes here? Why did we choose these numbers?

Reply:
I'm also curious why we feel we should override the default timeouts with lower values such as 10 and 5?

Reply:
In the distant past this step froze once and kept an agent occupied until it timed out around 1 hour. I wanted to prevent this... However it appears that issue was a one-off. I think it's ok to close this.